.github/workflows/gh-pages.yml

```diff
@@ -6,26 +6,40 @@ on:
 
 jobs:
 
-  documentation-job:
+  build-documentation:
     runs-on: ubuntu-24.04
-
+    permissions:
+      contents: read
     steps:
       - name: SCM Checkout
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
       - name: Setup Python & Poetry Environment
-        uses: exasol/python-toolbox/.github/actions/python-environment@1.0.1
+        uses: exasol/python-toolbox/.github/actions/python-environment@1.3.0
 
       - name: Build Documentation
         run: |
           poetry run -- nox -s docs:multiversion
+          rm -r .html-documentation/*/.doctrees
 
-      - name: Deploy
-        uses: JamesIves/github-pages-deploy-action@v4.7.2
+      - name: Upload artifact
```

Comment:
Updated the sqlalchemy-exasol per PTB 1.2.0 instructions so that pages per GitHub action (not branch)